sql/moz-fx-data-shared-prod/telemetry_derived/active_users_aggregates_attribution_v1/query.sql

```diff
@@ -50,130 +27,13 @@ WITH baseline AS (
-    MIN(submission_timestamp) AS min_submission_timestamp,
-    ARRAY_AGG(metrics.string.metrics_adjust_network)[SAFE_OFFSET(0)] AS adjust_network,
-    ARRAY_AGG(metrics.string.metrics_install_source)[SAFE_OFFSET(0)] AS install_source
-  FROM
-    org_mozilla_fennec_aurora.metrics AS org_mozilla_fennec_aurora_metrics
-  WHERE
-    DATE(submission_timestamp) >= '2021-01-01'
-    AND metrics.string.metrics_install_source IS NOT NULL
-    AND metrics.string.metrics_adjust_network IS NOT NULL
-  GROUP BY
-    client_id
-  UNION ALL
-  -- Fenix beta
-  SELECT
-    client_info.client_id AS client_id,
-    MIN(submission_timestamp) AS min_submission_timestamp,
-    ARRAY_AGG(metrics.string.metrics_adjust_network)[SAFE_OFFSET(0)] AS adjust_network,
-    ARRAY_AGG(metrics.string.metrics_install_source)[SAFE_OFFSET(0)] AS install_source
-  FROM
-    org_mozilla_firefox_beta.metrics AS org_mozilla_firefox_beta_metrics
-  WHERE
-    DATE(submission_timestamp) >= '2021-01-01'
-    AND metrics.string.metrics_install_source IS NOT NULL
-    AND metrics.string.metrics_adjust_network IS NOT NULL
-  GROUP BY
-    client_id
-  UNION ALL
-  -- Fenix release
-  SELECT
-    client_info.client_id AS client_id,
-    MIN(submission_timestamp) AS min_submission_timestamp,
-    ARRAY_AGG(metrics.string.metrics_adjust_network)[SAFE_OFFSET(0)] AS adjust_network,
-    ARRAY_AGG(metrics.string.metrics_install_source)[SAFE_OFFSET(0)] AS install_source
-  FROM
-    org_mozilla_firefox.metrics AS org_mozilla_firefox_metrics
-  WHERE
-    DATE(submission_timestamp) >= '2021-01-01'
-    AND metrics.string.metrics_install_source IS NOT NULL
-    AND metrics.string.metrics_adjust_network IS NOT NULL
-  GROUP BY
-    client_id
-),
-fenix_unique_attribution AS (
-  SELECT
-    client_id,
-    DATE(min_submission_timestamp) AS attribution_date,
-    ANY_VALUE(adjust_network) AS adjust_network,
-    ANY_VALUE(install_source) AS install_source
-  FROM
-    fenix_first_date_attributions
-  GROUP BY
-    client_id,
-    attribution_date
-),
-all_with_first_attribution AS (
-  SELECT
-    base.client_id,
-    base.attributed,
-    base.attribution_campaign,
-    base.attribution_content,
-    base.attribution_experiment,
-    base.attribution_medium,
-    base.attribution_source,
-    base.attribution_variation,
-    normalize_adjust_network(att.adjust_network) AS adjust_network,
-    normalize_install_source(att.install_source) AS install_source,
-    base.city,
-    base.country,
-    base.first_seen_date,
-    base.first_seen_year,
-    base.is_default_browser,
-    base.normalized_app_name,
-    base.submission_date,
-    base.days_since_seen,
-    base.ad_click,
-    base.organic_search_count,
-    base.search_count,
-    base.search_with_ads,
-    base.uri_count,
-    base.active_hours_sum
-  FROM
-    baseline AS base
+    `moz-fx-data-shared-prod.telemetry.unified_metrics` AS unified_metrics
   LEFT JOIN
-    fenix_unique_attribution AS att
+    fenix.firefox_android_clients AS att
```

Comment:
Raplacement to use `fenix.firefox_android_clients` to retrieve first unique attribution.

Reply:
It looks like the `fenix.firefox_android_client` table can have more than one row per `client_id`.  Therefore, this left join can result in more than one row per `client_id`.  Is this be the expected behaviour?

Reply:
Looking into this issue, we need to solve it from the upstream `moz-fx-data-shared-prod_mozilla_firefox_derived.baseline_clients_first_seen`, which potentially requires a backfill itself before we backfill `active_users_aggregates_attribution`. We don't require changes here, but in the related [Jira Task](https://mozilla-hub.atlassian.net/jira/software/c/projects/DENG/boards/465?modal=detail&selectedIssue=DENG-656).